schema/image-manifest-schema.json

```diff
@@ -14,12 +14,12 @@
       "$ref": "defs-image.json#/definitions/mediaType"
     },
     "config": {
-      "$ref": "defs-image.json#/definitions/descriptor"
+      "$ref": "content-descriptor.json"
     },
     "layers": {
       "type": "array",
       "items": {
-        "$ref": "defs-image.json#/definitions/descriptor"
+        "$ref": "content-descriptor.json"
```

Comment:
I reckon this is fine, but seems just as well to make it an object within a defs*.json file, rather than adding a new file just for a single definition.